selfdrive/ui/ui.cc

```diff
@@ -319,7 +322,7 @@ void QUIState::update() {
 
 Device::Device(QObject *parent) : brightness_filter(BACKLIGHT_OFFROAD, BACKLIGHT_TS, BACKLIGHT_DT), QObject(parent) {
   brightness_b = Params(true).get<float>("BRIGHTNESS_B").value_or(10.0);
-  brightness_m = Params(true).get<float>("BRIGHTNESS_M").value_or(0.1);
+  brightness_m = Params(true).get<float>("BRIGHTNESS_M").value_or(2.6) / 26.0;
```

Comment:
Old default was 2.6, and it was written to disk when the param was not present. We're stuk with this value.